cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -380,9 +380,11 @@ runTxBuildCmd
 
   txOuts <- mapM (toTxOutInAnyEra cEra) txouts
 
+  -- sort and filter out duplicates among the passed collateral inputs
```

Comment:
Can you change this to `-- the same collateral input can be used for several plutus scripts`. Also we don't need to sort the collateral inputs.

Reply:
Sure, done